src/Components/Footer/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import {
+  MDBFooter,
+  MDBContainer,
+  MDBRow,
+  MDBCol,
+  MDBIcon,
+} from "mdb-react-ui-kit";
+import footerlogo from "../../assets/footerlogo.jpeg";
+import { FooterLogoStyle, ContactStyle } from "./style";
+import { socialmediaIcon, contactInfo, templeTimes } from "./constant";
+
+export const Footer = () => {
+  return (
+    <MDBFooter className="text-center text-lg-start text-muted bg-dark ">
+      <section className="d-flex justify-content-center justify-content-lg-between p-4 border-bottom">
+        <div className="me-5 d-none d-lg-block">
+          <span>Get connected with us on social networks:</span>
+        </div>
+
+        <div>
+          {socialmediaIcon.map((item) => (
+            <a key={item.name} href={item.href} className="me-4 text-reset">
+              <MDBIcon color="secondary" fab icon={item.icon} />
+            </a>
+          ))}
+        </div>
+      </section>
+
+      <section className="d-flex justify-content-center justify-content-lg-between p-4 border-bottom">
+        <MDBContainer className="text-center text-md-start mt-5">
+          <MDBRow className="mt-3">
+            <MDBCol md="3" lg="4" xl="3" className="mx-auto mb-4">
+              <h6 className="text-uppercase fw-bold mb-4">
+                <MDBIcon color="secondary" icon="gem" className="me-3" />
+                Timings
+              </h6>
+              <div>
+                {templeTimes.map((time, index) => (
+                  <p key={index}>
+                    {time.name}: {time.time}
```

Comment:
```suggestion
                {schedules.map(schedule => (
                  <p key={schedule.name}>
                    {schedule.name}: {schedule.time}
```

Avoid using index as key wherever possible